README.md

```diff
@@ -3,53 +3,54 @@ o-tooltip [![Circle CI](https://circleci.com/gh/Financial-Times/o-tooltip/tree/m
 
 o-tooltip is a component usually used for annotating or highlighting bits of user interface.
 
-# TODO // todos here until v1 ships
-- Finish README
-- Tests
-- Calculation of the render postition
-- remove stuff duplicated stuff from o-overlay
-- pa11y demo
-
-
-_A table of contents to help people find things_
-
 - [Usage](#usage)
 	- [Markup](#markup)
 	- [JavaScript](#javascript)
 	- [Sass](#sass)
-- [Migration guide](#migration-guide)
 - [Contact](#contact)
 - [Licence](#licence)
 
 ## Usage
-o-tooltip has styles and JavaScript to show and hide a tooltip.
+o-tooltip has Sass and JavaScript to show and hide a tooltip which points at a target.
 
 ### Markup
 
 This HTML demonstrates the declarative way to instantiate o-tooltip. If you are using the Build Service or firing your own `o.DOMContentLoaded` event, this is all you need to create a tooltip.
 
 ```html
-TBC
+<div class='demo-tooltip-target' id="tooltip-target">
+	A bit of UI to annotate
+</div>
+
+<div data-o-component="o-tooltip"
+  data-o-tooltip-position="below"
+  data-o-tooltip-target='demo-tooltip-target'
+  data-o-tooltip-render-on-construction=true>
+	<div class='o-tooltip-content'>
+		Some text to go in the tooltip
+	</div>
+</div>
 
 ```
 
-A full list of data attributes:
+Attributes can be set declaratively, or passed in on instantiation in an options object. A full list of data attributes:
+- `data-o-tooltip-target`: Required. The target for the tooltip to point at
```

Comment:
Need to describe what type of value we are expecting, I think it is meant to be a CSS Selector?